doc/src/manual/variables-and-scoping.md

```diff
@@ -52,15 +55,21 @@ julia> Bar.foo()
 1
 ```
 
-Thus *lexical scope* means that the scope of variables can be inferred from the source code alone.
+Thus *lexical scope* means that what a variable in a particular piece of code refers to can be
+deduced from the code in which it appears alone and does not depend on how the program executes. A
+scope nested inside another scope can "see" variables in all the outer scopes in which it is
+contained. Outer scopes, on the other hand, cannot see variables in inner scopes.
 
 ## Global Scope
 
-Each module introduces a new global scope, separate from the global scope of all other modules;
-there is no all-encompassing global scope. Modules can introduce variables of other modules into
-their scope through the [using or import](@ref modules) statements or through qualified access using the
-dot-notation, i.e. each module is a so-called *namespace*. Note that variable bindings can only
-be changed within their global scope and not from an outside module.
+Each module introduces a new global scope, separate from the global scope of all other modules—there
+is no all-encompassing global scope. Modules can introduce variables of other modules into their
+scope through the [using or import](@ref modules) statements or through qualified access using the
+dot-notation, i.e. each module is a so-called *namespace* which means it is a first-class data
```

Comment:
I don't think "namespace" implies first-class-ness; for example C++ namespaces are not first class.

Reply:
Aight, I was just trying to make more sense of what was here previously.